vxfreeswitch/voice.py

```diff
@@ -336,10 +341,17 @@ def register_client(self, client):
 
     def deregister_client(self, client, duration=None):
         client_addr = client.get_address()
+
+        # If originated call has not yet been answered
+        if self._unanswered_channels.get(client_addr):
+            d = self._unanswered_channels[client_addr]
+            d.errback(FreeSwitchClientError('Call is unanswered'))
```

Comment:
We will always be sending an outbound and I see the problem with `handle_outbound_message` / `dial_outbound`.